js/ui/components/codingGameService.js

```diff
@@ -24,6 +24,33 @@ const CodingChatboxTextService = new Lang.Class({
         }
     },
 
+    _updateListeningForEvents: function() {
+        if (!this._service) {
```

Comment:
Don't think this can really happen, can it? Otherwise we would have already errored out earlier